mmv1/products/securesourcemanager/Repository.yaml

```diff
@@ -0,0 +1,170 @@
+# # Copyright 2024 Google Inc.
```

Comment:
Is this file auto-generated based on our API proto?

Reply:
No I wrote it manually

Reply:
But since I wrote the Instance.yaml, for this one I mostly just copied that one over